AU2/plugins/CorePlugin.py

```diff
@@ -381,6 +385,21 @@ def answer_core_plugin_update_assassin(self, html_response_args: Dict):
             return_components += p.on_assassin_update(assassin, html_response_args)
         return return_components
 
+    def ask_core_plugin_hide_assassins(self):
+        assassins = ASSASSINS_DATABASE.get_identifiers(include_hidden=True)
+        components = [SelectorList(identifier=self.identifier+"_hidden_assassins",

```

Comment:
This identifier should probably live in HTML ids

Reply:
I was thinking about doing this originally but I was concerned by the fact that `html_ids` only includes the identifiers for Assassin-relevant components, and the other identifiers are segregated into seperate dicts (`event_html_ids`, `config_html_ids`).

Relatedly, I've been wondering what the purpose of `html_ids_list` is? It doesn't seem to actually be used anywhere and currently just duplicates `html_ids.keys()`.

Reply:
There's a good chance this is tech debt built up during the rush to get AU2 finished before MW24. Feel free to delete if there's no need for it.